include/nbl/builtin/hlsl/matrix_utils/matrix_traits.hlsl

```diff
@@ -16,7 +16,8 @@ struct matrix_traits;
 template<typename T> \
 struct matrix_traits<matrix<T, ROW_COUNT, COLUMN_COUNT> > \
 { \
-    using ComponentType = T; \
+    using ScalarType = T; \
+    using RowType = vector<T, COLUMN_COUNT>; \
```

Comment:
use snake case for these typedefs, and camel case for concepts and template parameters like `std::` does